packages/ndla-ui/src/Figure/Figure.tsx

```diff
@@ -35,57 +35,108 @@ export const FigureCaption = ({
   link,
   hideFigcaption,
   hasLinkedVideo,
+  hideIconsAndAuthors,
 }: FigureCaptionProps) => {
   const { t } = useTranslation();
   return (
-    <figcaption {...classes('caption', hideFigcaption ? 'hidden-caption' : undefined)}>
-      {caption ? <div {...classes('info')}>{parseMarkdown(caption)}</div> : null}
-      <footer {...classes('byline')}>
-        <div {...classes('byline-licenselist')}>
-          <LicenseByline licenseRights={licenseRights} locale={locale} marginRight>
-            <div {...classes('byline-author-buttons')}>
-              <span {...classes('byline-authors')}>{authors?.map((author) => author.name).join(', ')}</span>
-              <div>
-                <Button
-                  borderShape="rounded"
-                  outline
-                  size="small"
-                  type="button"
-                  data-dialog-trigger-id={id}
-                  data-dialog-source-id={figureId}>
-                  {reuseLabel}
-                </Button>
-                {hasLinkedVideo && (
+    <>
+      {!hideIconsAndAuthors ? (
```

Comment:
Veldig masse duplisering her. Treng du alt det, eller kan du heller bruke hideIconsAndAuthors for å skjule kun det?